discounts/rust/product-discounts/default/src/main.rs

```diff
@@ -3,273 +3,75 @@ use serde::{Deserialize, Serialize};
 mod api;
 use api::*;
 
-#[derive(Clone, Debug, Deserialize)]
-pub struct Payload {
-    pub input: input::Input,
-    pub configuration: Configuration,
-}
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ConfigurationValue {}
 
-#[derive(Clone, Debug, Deserialize)]
-#[serde(rename_all(deserialize = "camelCase"))]
-pub struct Configuration {
-    pub value: Option<String>,
-    pub excluded_variant_ids: Option<Vec<ID>>,
-}
+#[derive(Clone, Debug, Serialize, Deserialize, Default)]
```

Comment:
Why is `Serialize` needed?

Reply:
Prob not needed, the test only serializes `ConfigurationValue` now, I'll remove it from this

Reply:
Using it to serialize the Configuration in the test now